translator/source_code/icone_translator.py

```diff
@@ -7,6 +7,26 @@
 from translator.source_code import translator_shared_library
 
 
+# Features
```

Comment:
Same as above, I think we can remove these comments.

Reply:
removed those moments